src/middleware.ts

```diff
@@ -46,7 +46,7 @@ export function build(
   conf: SiberConfig = {}
 ) {
   // default middleware
-  app.use(express.json());
+  app.use(express.json({ limit: conf.requestBodyLimit }));
   app.use(express.urlencoded({ extended: false }));
```

Comment:
`requestBodyLimit` should also affect `URLEncoded` requests. The name doesn't distinguish the type of request.

Reply:
Alright. could this be a better name `requestPostLimit`?

Reply:
I'm not talking about a name change. I'm saying the name implies it affects the body regardless of the type so it means both `JSON` and `URLEncoded` body types should be affected